voidcrew/edits/mobs/new_player.dm

```diff
@@ -11,7 +11,7 @@
 			CRASH("[active_ships] has no shuttle???")
 		if(length(active_ships.shuttle.spawn_points) <= 0)
 			continue
-		shuttle_choices["[active_ships.name]" - "([active_ships.source_template.short_name])"] = active_ships
+		shuttle_choices["[active_ships.name] - ([active_ships.source_template?.short_name || "Unknown Class"])"] = active_ships
```

Comment:
There should always have a short name

Reply:
not if they are player created

Reply:
What do you mean? When does this get removed?

Reply:
Supposedly you can create a ship from scratch entirely

Reply:
That is fucked up what the hell